x-pack/plugins/cloud_security_posture/public/components/chart_panel.tsx

```diff
@@ -63,8 +63,8 @@ export const ChartPanel: React.FC<ChartPanelProps> = ({
       <EuiFlexGroup direction="column" gutterSize="none" style={{ height: '100%' }}>
         <EuiFlexItem grow={false}>
           {title && (
-            <EuiTitle size="s">
-              <h3>{title}</h3>
+            <EuiTitle size="xs">
+              <h3 style={{ lineHeight: 'initial' }}>{title}</h3>
```

Comment:
Why do you set `line-height`?

Reply:
yeah forgot to comment about that, the cluster card's `title - id` header is now a EuiLink, which changed the line height for this title. made the titles for the cluster card and the other panels to be in slightly different heights. looked for ways around that but eventually, this was the only solution that worked.